src/pages/sign-in/sign-in.tsx

```diff
@@ -2,34 +2,32 @@ import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
 import { Avatar, Checkbox, Container, FormControlLabel, Typography } from '@mui/material';
 import { Field, Formik } from 'formik';
 import { TextField } from 'formik-mui';
-import React, { useEffect, ReactText, ReactElement } from 'react';
+import React, { ReactElement, ReactText } from 'react';
 import { useNavigate } from 'react-router-dom';
 import { ToastContainer, toast } from 'react-toastify';
 
 import Copyright from '../../components/copyrigth/copyrigth';
 import { APP_ROUTES } from '../../enums';
-import { useAppDispatch, useAppSelector } from '../../hooks';
+import { useAppSelector } from '../../hooks';
 import { User } from '../../interfaces';
-import { loadUsers, usersSelector } from '../../store/users.slices';
+import { usersSelector } from '../../store/users.slices';
+import UserAuth from '../../components/auth/user-auth';
 import { FlexColumn, FlexCenter, SignInButton } from './sign-in.styles';
 import validationSchema from './validationSchema';
 
 const Signin = (): ReactElement => {
   const navigate = useNavigate();
 
-  const dispatch = useAppDispatch();
   const users: User[] = useAppSelector(usersSelector.selectAll);
 
-  useEffect(() => {
-    dispatch(loadUsers());
-  }, [dispatch]);
-
   const initialValues: User = {
     id: '',
     username: '',
     password: '',
   };
 
+  const userAuth = new UserAuth();
```

Comment:
create variable inside file with class `UserAuth` and just export it, so we will have one instance of this service for whole app